### DISCUSSION TITLE
Kirkstone Jetson Orin Nano extend UDA partition size

### DISCUSSION THREAD

**Discussion by anhhao17** [OE4T/meta-tegra#1633] [Q&A]
Hi guys,
I'm new to working with Yocto and I have a question regarding disk partitioning. Specifically, I'm interested in learning how to extend the UDA partition to a specific size or alternatively, how to configure it to utilize all the remaining space on the disk.

Thank you for your assistance!

**Comment by dwalkes** [comment]
Hi @anhhao17 and welcome!

You can add a custom partition layout using the instructions in https://github.com/OE4T/meta-tegra/discussions/1467 - note that the process varies a bit by branch.  If you want to help update documentation when you go through this that would be much appreciated.  We could use an update at https://github.com/OE4T/meta-tegra/wiki/Redundant-Rootfs-A-B-Partition-Support or a page linked from there.

There's also an example which modifies the UDA partition in a partition layout provided by NVIDIA in https://github.com/OE4T/meta-mender-community/pull/19/files#r1676839287.
